pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
     "craft-application>=4.0.0",
     "craft-archives>=2.0.0",
     "craft-cli",
-    "craft-parts>=2.1.2",
+    "craft-parts>=2.1.2,!=2.1.3",
```

Comment:
I kind of want to "force our hands" to get this fixed sooner rather than later; if we pin this to `==2.1.2` we might just forget to ever update craft-parts again